lib/rules/id-helper.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  meta: {
+    type: 'problem',
+    fixable: false,
+    docs: {
+      description: 'Use `idHelper` for inserts in repositories',
+      category: 'Error',
+      recommended: true
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {},
+        additionalProperties: false
+      }
+    ],
+    messages: {
+      useIdHelper: 'Use idHelper for `insert`, `insertAndReturnFirstRow` and `bulkCreate`'
+    }
+  },
+
+  create: function (context) {
+    return {
+      'CallExpression[callee.property][callee.object.name!="dbBatch"][callee.property.name=/(insert|insertAndReturnFirstRow)/]'(
```

Comment:
I'm a little confused about the inclusion of `insertAndReturnFirstRow` - aren't calls to that function already wrapped via https://github.com/thoughtindustries/ti/commit/8274cb8dc615103b10dd54b23fcdee0f97ecbf13#diff-e022be96c8d8394b38b4498114410b7e0a5af8308d279f928bf6639e9e4eb5d3R96 ?

Reply:
Removed!